src/js/front-end/actions/keybinds/keybinds.js

```diff
@@ -78,6 +82,9 @@ export const keyDown = (event) => {
             shuffleBoard(systemState.initiator, systemState.initiator);
         };
         if (event.key === 'f' && !(event.altKey || event.getModifierState('Alt'))) {
+            if(systemState.isReplay && !systemState.isTwoPlayer){
```

Comment:
instead of writing this a bunch of times for all the respective keybinds, let's put all of the relevant keybinds nested in one if statement (or move all of the ones that aren't impacted to the top, and then afterwards have an if statement that returns early)

Reply:
I moved the unaffected ones to the top in [9588910](https://github.com/xxmichaellong/ptcg-sim/pull/19/commits/9588910523e901a643f28aa12aad8b81cf303b6f). Had to split an if statement to do that, but I thought it was the cleanest way possible